standards/SPIFFE_Trust_Domain_and_Bundle.md

```diff
@@ -0,0 +1,290 @@
+# The SPIFFE Trust Domain and Bundle
```

Comment:
In the course of porting this document to markdown, I realized that none of the SPIFFE specs have authors listed. I will submit another PR once this one is in, retroactively crediting authors involved in each of the SPIFFE documents